### PR TITLE
feat(video-conferencing-service): SFO-103,SFO-107 implement GET attendee list, allow webhook to handle more events

### PR DESCRIPTION
**Pull request by anitsingh-sf**
SFO-103

## Description

 SFO-103 implement GET attendee list

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Intermediate change (work in progress)

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

## Checklist:

- [x] Performed a self-review of my own code
- [x] npm test passes on your machine
- [x] New tests added or existing tests modified to cover all changes
- [x] Code conforms with the style guide
- [ ] API Documentation in code was updated
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: services/video-conferencing-service/src/controllers/video-chat-session.controller.ts [hunk 1]
```diff
@@ -325,4 +326,59 @@ export class VideoChatSessionController {
...
+    });
+
+    if (!videoSessionDetail) {
+      errorMessage = 'Meeting Not Found';
+      auditLogPayload.after = {errorMessage};
+      await this.auditLogRepository.create(auditLogPayload);
+      throw new HttpErrors.NotFound(errorMessage);
+    }
+
+    if (videoSessionDetail.endTime) {
```
Comment (bhavyasf) [inline services/video-conferencing-service/src/controllers/video-chat-session.controller.ts:366]: @anitsingh-sf  remove this

Comment (anitsingh-sf) [reply services/video-conferencing-service/src/controllers/video-chat-session.controller.ts]: done

File: services/video-conferencing-service/src/controllers/video-chat-session.controller.ts [hunk 2]
```diff
@@ -325,4 +364,64 @@ export class VideoChatSessionController {
       );
     }
   }
+
+  @authenticate(STRATEGY.BEARER)
+  @authorize([PermissionKeys.GetAttendees])
+  @get('/session/{meetingLinkId}/attendees', {
+    parameters: [{name: 'areActive', schema: {type: 'string'}, in: 'query'}],
```
Comment (bhavyasf) [inline services/video-conferencing-service/src/controllers/video-chat-session.controller.ts:373]: replce 'areActive' with simple 'active'

Comment (anitsingh-sf) [reply services/video-conferencing-service/src/controllers/video-chat-session.controller.ts:373]: ok

File: services/video-conferencing-service/src/controllers/video-chat-session.controller.ts [hunk 3]
```diff
@@ -275,27 +275,68 @@ export class VideoChatSessionController {
...
-              modifiedOn: new Date(),
-            },
+            updatedAttendee,
+          );
+        }
+
+        if (event === VonageEnums.SessionWebhookEvents.StreamCreated) {
+          await this.sessionAttendeesRepository.updateById(
+            sessionAttendeeDetail.id,
+            updatedAttendee,
```
Comment (bhavyasf) [inline services/video-conferencing-service/src/controllers/video-chat-session.controller.ts:313]: also update extMetaData

File: services/video-conferencing-service/src/controllers/video-chat-session.controller.ts [hunk 4]
```diff
@@ -275,27 +275,68 @@ export class VideoChatSessionController {
...
+          extMetadata: {webhookPayload: webhookPayload},
+        };
+        if (event === VonageEnums.SessionWebhookEvents.ConnectionCreated) {
+          updatedAttendee.isDeleted = false;
           await this.sessionAttendeesRepository.updateById(
             sessionAttendeeDetail.id,
-            {
-              modifiedOn: new Date(),
-            },
+            updatedAttendee,
```
Comment (bhavyasf) [inline services/video-conferencing-service/src/controllers/video-chat-session.controller.ts:306]: update extMetadata

**Pull request merged**